src/nub.ts

```diff
@@ -292,8 +244,9 @@ export class NUB {
   public async estimateGasForTokenTransfer(tokenAddress: string, receiver: string, amount: number) {
     const contract = new this.web3.eth.Contract(Abi.basics.erc20, tokenAddress);
     const from = await this.internal.getAddress();
-    const gas = await contract.methods.transfer(receiver, amount).estimateGas({ gas: this.GAS_PRICE, from });
-    return gas;
+    console.log(contract, this.GAS_PRICE, from);
```

Comment:
@nonseodion, did you leave this log here on purpose?

Reply:
Nope I didn't thanks for pointing it out. Fixed it.